src/exports.h

```diff
@@ -11,6 +11,10 @@
 #define TJFLAG_FASTDCT 0
 #endif
 
+#ifdef DEFAULT_QUALITY
+#undef DEFAULT_QUALITY
```

Comment:
Perhaps we would prefix our defines instead of doing this. Think you could update the code?